tests/edit_inputs.sh

```diff
@@ -52,11 +52,12 @@ function edit_data_table {
 }
 
 function edit_diag_table {
-  sed -e "s/YMD/$SYEAR$SMONTH$SDAY/g" \
-      -e "s/ATMRES/$ATMRES/g" \
-      -e "s/SYEAR/$SYEAR/g" \
-      -e "s/SMONTH/$SMONTH/g" \
-      -e "s/SDAY/$SDAY/g"
+  sed -e "s/@[YMD]/$SYEAR$SMONTH$SDAY/g" \
+      -e "s/@[ATMRES]/$ATMRES/g" \
+      -e "s/@[SYEAR]/$SYEAR/g" \
+      -e "s/@[SMONTH]/$SMONTH/g" \
+      -e "s/@[SDAY]/$SDAY/g" \
+      -e "s/@[SHOUR]/$SHOUR/g"
 }
```

Comment:
This function is not necessary anymore. We should use atparse.

Reply:
Would that mean replace:

```
edit_diag_table < ${PATHRT}/parm/${DIAG_TABLE:-diag_table_template} > diag_table
```
wtih
```
atparse < ${PATHRT}/parm/${DIAG_TABLE:-diag_table_template} > diag_table
```
on [L123](https://github.com/aerorahul/ufs-weather-model/blob/1e48750fb1d853c09c74616c1bfad086dea63e6e/tests/run_test.sh#L123) of `run_test.sh`?

Reply:
That is correct. We are going to replace all the edit_ functions with atparse. Would you please make this change, then we can commit it? Thanks